dataeng/jobs/analytics/WarehouseTransforms.groovy

```diff
@@ -55,4 +60,3 @@ class WarehouseTransforms{
         }
     }
 }
-
```

Comment:
This looks strange. There were two blank lines at the end of this file. I removed one, but it looks like there are none now? Either way, if it matters, there should still be one blank line hah.